### DISCUSSION TITLE
Support for Banners in Catalyst

### DISCUSSION THREAD

**Discussion by Cliftonz** [bigcommerce/catalyst#1699] [Feature Requests]
Introducing native support for customizable banners in Catalyst would enable merchants to effectively showcase promotions, announcements, and key information, thereby enhancing user engagement and driving conversions.

Proposed Features:
- Customizable Banner Component: A reusable component that can be seamlessly integrated across various storefront pages.
- Integration with Admin: The customizable component shows up with the settings in the banner section of the admin page.

**Comment by bookernath** [comment]
@Cliftonz our solution for these sorts of things is a deep integration with Makeswift; if you have a partner sandbox, you can test this today in your control panel by creating a new Catalyst channel from the Channel Manager.

It goes far beyond banners :)

Users of Catalyst can of course use any headless CMS or visual editor they like, but Makeswift is an included option.

**Reply by Cliftonz** [reply]
@bookernath Thank you for the explanation.

I think this should go in the docs though as I feel like a lot of people will have the same question.
